packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/__tests__/barrier-selector.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import BarrierSelector from '../barrier-selector';
+import TradeParams from '../../../../../Containers/trade-params';
+
+const mock_props = {
+    barrier_1: '16',
+    onChange: () => {
+        mock_props.barrier_1 = '33';
```

Comment:
I tried to refer with `this` as well as a classic function (not arrow), but in both cases get `undefined` instead of `barrier_1`. That was the only option that worked.

Reply:
@kate-deriv let's replace it with jest.fn() and we'll test our component based on incoming barrier_1 prop.
`onChange: jest.fn()`

Reply:
Done, thank you!